docs/sources/flow/reference/components/prometheus.exporter.mysql.md

```diff
@@ -0,0 +1,141 @@
+`enable_collectors`                               | `list(string)` | A list of [collectors][] to enable on top of the default set.  | | no
+`disable_collectors`                              | `list(string)` | A list of [collectors][] to disable from the default set. | | no
+`set_collectors`                                  | `list(string)` | A list of [collectors][] to run. Fully overrides the default set. | | no
+`lock_wait_timeout`                               | `int`          | Timeout, in seconds, to acquire a metadata lock. | `2`| no
+`log_slow_filter`                                 | `bool`         | Used to avoid queries from scrapes being logged in the slow query log. | `false` | no
+`info_schema_processlist_min_time`                | `int`          | Minimum time a thread must be in each state to be counted. | `0` | no
+`info_schema_processlist_processes_by_user`       | `bool`         | Enable collecting the number of processes by user. | `true` | no
+`info_schema_processlist_processes_by_host`       | `bool`         | Enable collecting the number of processes by host. | `true` | no
+`info_schema_tables_databases`                    | `string`       | Regular expression to match databases to collect table stats for. | `"*"` | no
+`perf_schema_eventsstatements_limit`              | `int`          | Limit the number of events statements digests, in descending order by `last_seen`. | `250` | no
+`perf_schema_eventsstatements_time_limit`         | `int`          | Limit how old, in seconds, the `last_seen` events statements can be. | `86400` | no
+`perf_schema_eventsstatements_digtext_text_limit` | `int`          | Maximum length of the normalized statement text. | `120` | no
+`perf_schema_file_instances_filter`               | `string`       | Regular expression to select rows in `performance_schema.file_summary_by_instance`. | `".*"` | no
+`perf_schema_file_instances_remove_prefix`        | `string`       | Prefix to trim away from `file_name`.  | `"/var/lib/mysql"` | no
+`heartbeat_database`                              | `string`       | Database to collect heartbeat data from. | `"heartbeat"` | no
+`heartbeat_table`                                 | `string`       | Table to collect heartbeat data from. | `"heartbeat"` | no
+`heartbeat_utc`                                   | `bool`         | Use UTC for timestamps of the current server (`pt-heartbeat` is called with `--utc`). | `false` | no
+`mysql_user_privileges`                           | `bool`         | Enable collecting user privileges from mysql.user. | `false` | no
+
+
+Set a `lock_wait_timeout` on the connection to avoid potentially long wait times for metadata locks. View more detailed documentation on `lock_wait_timeout` [here](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_lock_wait_timeout).
```

Comment:
Try to avoid links that just say "here" and give a short description for where you're leading people:

```suggestion
Set a `lock_wait_timeout` on the connection to avoid potentially long wait times for metadata locks. View more detailed documentation on `lock_wait_timeout` [in the MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_lock_wait_timeout).
```